slitlessutils/examples/wfc3ir.py

```diff
@@ -190,8 +190,8 @@ def plot_spectra():
 
     # get the sensitivity curve just for fun
     subpath = os.path.join('instruments', 'WFC3IR')
-    senfile = cfg.get_reffile('WFC3.IR.G102.1st.sens.2.fits', subpath)
-    sens = fits.getdata(senfile)
+    sendfile = cfg.get_reffile('WFC3.IR.G102.1st.sens.2.fits', subpath)
+    sens = fits.getdata(sendfile)
```

Comment:
`codespell` (called by `pre-commit`) thinks "senfile" is a typo.  Maybe rename to "sens_file"?  

We could also ignore this by adding a section in pyproject.toml:

```
[tool.codespell]
ignore-words-list = """
    senfile,
"""
```

Reply:
Actually, this is the only place it's used, I'll just use `sens_file`.